src/sentry/api/endpoints/organization_releases.py

```diff
@@ -161,6 +162,11 @@ def post(self, request, organization):
             if commit_list:
                 release.set_commits(commit_list)
 
+            head_commits = result.get('head_commits')
+            if head_commits:
+                fetch_commits = request.user.is_authenticated() and not commit_list
+                release.set_head_commits(head_commits, request.user, fetch_commits=fetch_commits)
```

Comment:
i'm not sure how we wanna handle failure here. do we just create the release and say everything is ok? or do we let people know we couldn't find commits?